content/capture-app/app-android-settings-configuration.md

```diff
@@ -256,14 +256,24 @@ It allows the admin user to enable or disable the option to show the filters rel
 
 ### Program { #capture_app_android_settings_webapp_appearance_program }
 
-It allows the admin user to specify which filters should be shown and enable or disable the option to show the completion percentage of the data entry form.
+It allows the admin user to specify which filters should be shown.
```

Comment:
I think this could introduce to both global and specific settings:

"Program appearance allows to hide/show features within the program according to the configuration needs. These changes can happen globally, applying to ALL the programs, or specifically to a single one."